components/google-cloud/tests/container/experimental/notebooks/test_executor.py

```diff
@@ -0,0 +1,144 @@
+import json
+from logging import raiseExceptions
+import os
+import time
+import pytest
+import unittest
+from unittest import mock
+from google.cloud import notebooks
+from google.api_core import operation
+from google.cloud.notebooks import Execution
+from google.cloud import aiplatform as vertex_ai
+from google.cloud.aiplatform import CustomJob
+from google.cloud import aiplatform_v1beta1 as vertex_ai_beta
+from google.cloud.aiplatform.compat.types import job_state
+from types import SimpleNamespace
+
+from google_cloud_pipeline_components.container.experimental.notebooks import executor
+
+_MOCK_PROJECT_ID = 'mock-project-id'
```

Comment:
@IronPan  should we standardize the unit tests to follow one paradigm only. Most tests currently use unittest however there are two including this cl that are using pytest. This is not a major issue however it may add complexity in maintaining the infra in the future.

Reply:
If I am not to use pytest, what would be the recommended library for parametrization? I see some code using `from absl.testing import parameterized`.  Or maybe unittest has an option but I am not aware of it.